roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -27,6 +24,10 @@ class DockerImageAvailability(OpenShiftCheck):
             "repo": "openshift3",
             "image": "ose",
         },
+        "enterprise": {
+            "repo": "openshift3",
+            "image": "ose",
+        },
```

Comment:
As per recent discussion and documentation update in #3799, `enterprise` is not in use at the moment, and was used for 3.0.x.

@brenton do you think checks should go as far as supporting OSE 3.0?

In this case I'd probably have support for only `origin` and `openshift-enterprise` (for OSE/OCP 3.1+).

If need be, for this or other checks, we could also explicitly disable the checks based on the target version... but in practice the format of the inventory we expect is a current one, we don't go back to support what might an inventory have looked like back in 3.0..., so it might even be a challenge to handle incompatibilities in inventories -- if there was ever any backward-incompatible change in name of variables for example (and I am aware of a recent change from deployment_type -> openshift_deployment_type).